src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -174,7 +195,7 @@ export const getWorkflowStepsForInteraction = (
     status: string;
   };
 } => {
-  return WorkflowStepsForInteractions[interaction];
+  return _.cloneDeep(WorkflowStepsForInteractions[interaction]);
```

Comment:
`structuredClone()` is a built-in function for deep-copying. lodash is a large package, so avoid it if we can

https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

Reply:
added